src/main/java/se/sundsvall/emailreader/api/EmailResource.java

```diff
@@ -0,0 +1,106 @@
+package se.sundsvall.emailreader.api;
+
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_PROBLEM_JSON_VALUE;
+
+import java.util.List;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.zalando.problem.Problem;
+
+import se.sundsvall.emailreader.api.model.Email;
+import se.sundsvall.emailreader.service.EmailService;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+
+@RestController
+@RequestMapping(path = "/email",
+    consumes = APPLICATION_JSON_VALUE,
```

Comment:
Shouldn't this be defined at method level instead? If this remains defined at class level, it will be applicable for all methods.

GET-methods doesn't consume anything (at least not if you follow REST-conventions).